### PR TITLE
The next web home page clone

### PR DESCRIPTION
**Pull request by mohapakram**
In this home page I: 
* Aligned all elements with float, flex, and grid.
* Used a mobile-first approach when dealing with media queries with 2 breakpoints.
* Made it fully responsive and tested it.

### CODE REVIEW

**Review by expjazz — CHANGES_REQUESTED** [review]
## Hey team,

### [Changes Required] 

##### Good work on this project, it looks great! However, there are still some changes we need to do to get this project approved. I will divide in sections:

#### Navbar
 
- [x] TWN logo SHOULD appear in the leftmost part in the medium screen.

#### Original
:point_down: 

Screen Shot 2020-07-09 at 14 49 56

#### Your clone

:point_down: 
Screen Shot 2020-07-09 at 14 51 25

#### Top three section

- [x] On the medium screen the TNW logo should disappear from this section. The orange gradient also needs to disappear. Pay attention to the font sizes on smaller screen sizes (it seems to be overflowing).

#### Original
:arrow_down: 

Screen Shot 2020-07-09 at 15 00 46

#### Your clone
:arrow_down: 
Screen Shot 2020-07-09 at 15 01 29

- [x] On the mobile screen the text is too big and ends up being unreadable. 
:arrow_down: 
Screen Shot 2020-07-09 at 15 04 54

### Latest news section

- [x] It should have three articles per row in the medium screen, not four.

#### Original
:arrow_down: 
Screen Shot 2020-07-09 at 15 08 35
#### Yours
:arrow_down: 
Screen Shot 2020-07-09 at 15 09 32

### Latest funding section

- [x] As in the sections above, it appears you ignored the medium screen size. It also should have only three columns per row, not four

### Categories section

- [x] Again, it appears you forgot about the medium screen. Kindly fix that (should be 2 category boxes per row).

### Latest deals 

- [x] Please check the medium screen size.

### Footer

- [x] Make your elements smaller on the mobile screens. At the moment they are not responding well on smaller screen sizes

#### Your project looks quite good on the large screen. Please work or the middle one and then submit for another review once you are ready.

[odin screenshot]
image

Keep up the good work! :smile: 

Good Luck and Happy Coding :pray:

P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade). :handshake:

File: styles.css [hunk 1]
```diff
@@ -57,8 +67,501 @@ ul a {
...
+
+#main-nav ul li {
+  margin-right: 35px;
+  list-style-type: none;
+  padding: 10px 0;
+}
+
+#main-nav a,
+#header-icons i {
+  color: #aaa;
```
Comment (bertil291utn) [inline styles.css]: ### Using the same Rule repetitively. 

Avoid repeat same rule `color: #d33a2c;` tons of time. Use  instead `.gray{color: #d33a2c;}`

File: index.html [hunk 2]
```diff
@@ -27,15 +44,712 @@
...
+         <div>
+            <h3> <a href="#">New AI project captures Jane Austen's thoughts on social media</a></h3>
+            <div class="article-meta">
+              <span class="author"><a href="#">Thomas Macaulay</a></span>
+              <span>.</span>
+            
+              <span class="time">3 hours ago</span>
+            </div>
+         </div>
+        </article><article class="article-item flex-container">
```
Comment (bertil291utn) [inline index.html]: ## Html semantics
Wrap your articles or any section ith `<section>` tag

**Review by bertil291utn — CHANGES_REQUESTED** [review]
**Status:** Changes requested 🔴  :shipit: ? Not yet

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.

Your work is amazing you master it GitHub Flow you have a professional README file.

The webpage looks amazing it's clean and neat, however, you have to make some changes.

**Medium screen**

- [x] Your entire web page doesn't have a medium screen approach, you have to complete it.

image



 ### CSS practices

- [ ] **Using the same Rule repetitively.** Avoid repeat same rule `color: #d33a2c;` tons of time. Use  instead `.gray{color: #d33a2c;}`
- [ ] **Over Using the !important Rule.** Using this out of laziness will give you a headache later on as your style sheet will be harder to maintain.

```css
 #featured-blocks h1 {
    font-size: 20px !important;
    line-height: 31px !important;
    margin: 10px 0;
  }
```

- [ ] Remove unnecessary declared rules

```css
/* header{
} */
```



<hr>

Guys work in the requested changes above and request another code review.
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

File: styles.css (hunk 1 above)
Comment (bertil291utn) [reply styles.css]:
```
color: #aaa;

File: index.html (hunk 2 above)
Comment (mohapakram) [reply index.html]: `<section>` tags give an error in the HTML validator because they should have an `<h1>`  as a direct child.

**Comment by ikraamg** [comment]
### **Status: PROJECT APPROVED**  :1st_place_medal: :1st_place_medal: :1st_place_medal:


Hi @mohapakram  :smile:  Congratulations, your project looks amazing and you did an amazing job improving it and making the neccesary changes :sunglasses: 

You can submit your project! 

**A little hint that is sometimes useful:** 
When you want to stop your site from breaking when too small or looking odd on big screens, you can apply the following css code on your **html or body** tag of your page:
_min-width: 350px;_ (you can pick your pixels)
_max-width: 1366px;_ (you can pick your pixels)

If you have any questions, don't hesitate to contact me through slack [@ikraam](url)


_**Happy coding and good luck on the next project**_  :desktop_computer: 

_Project reviewed by Ikraam Ghoor (TSE)_

**Pull request merged**